packages/odyssey-design-tokens/src/color/palette.json

```diff
@@ -8,105 +8,125 @@
         },
         "000": {
           "comment": "Used for Backgrounds, read-only & disabled input fills",
-          "value": "#f5f5f6"
+          "value": "#F4F4F4"
         },
         "100": {
           "comment": "Used for read-only/disabled states",
-          "value": "#ebebed"
+          "value": "#EDEDED"
+        },
+        "200": {
+          "comment": "Used for Display borders",
+          "value": "#E1E1E1"
         },
-        "200": { "comment": "Used for Display borders", "value": "#d7d7dc" },
```

Comment:
Figma has a Grey 300 as well, does that not exist in code currently?

Reply:
Correct, there's no `300` in the current tokens.